lib/rules/function-name-case/index.js

```diff
@@ -45,7 +45,6 @@ const rule = function(expectation, options, context) {
     }
 
     root.walkDecls(decl => {
-      let needFix = false;
```

Comment:
there is some performance idea in this variable ( `parsed.toString()` is a bit expensive operation, see [stringify.js](https://github.com/TrySound/postcss-value-parser/blob/master/lib/stringify.js)), for instance when function name is correct, but `context.fix === true`, it is not necessary to perform `parsed.toString()`